processor/cumulativetodeltaprocessor/config.go

```diff
@@ -52,5 +62,26 @@ func (config *Config) Validate() error {
 		(len(config.Exclude.MatchType) > 0 && len(config.Exclude.Metrics) == 0) {
 		return fmt.Errorf("metrics must be supplied if match_type is set")
 	}
+
+	for i, metricType := range config.Exclude.MetricTypes {
+		config.Exclude.MetricTypes[i] = strings.ToLower(metricType)
```

Comment:
Lets not change any values as part of the Validate. a conversion from these strings to a map that uses the pmetric.MetricType enum can be done in the processor constructor.